src/Styleguide/Artwork/Banner.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import styled from "styled-components"
+import { Col, Row } from "../Elements/Grid"
+
+const Flex = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+interface RoundedImageProps {
+  size: string
+}
+
+const RoundedImage = styled.img.attrs<RoundedImageProps>({})`
+  width: ${(props: RoundedImageProps) => props.size};
+  height: ${props => props.size};
+  border-radius: ${props => props.size};
+`
+
+export class Banner extends React.Component {
+  render() {
+    return (
+      <Row>
+        <Col sm={8}>
+          <Flex>
+            <RoundedImage
+              size="110px"
+              src="https://picsum.photos/110/110/?random"
```

Comment:
Wondering if/where/how we would deal with possibility of image is not present here?

Not sure it's a very likely scenario in this case, but it will be very much likely if we display gallery logo/artist image in  this avatar style.